app/views/user_mailer/in_person_please_call.html.erb

```diff
@@ -6,9 +6,10 @@
     ) %>
 <h1><%= t('user_mailer.in_person_please_call.header') %></h1>
 <p>
+  <%= p locale %>
```

Comment:
this looks like it's for debugging? I would remove
```suggestion
```

Reply:
oh yes thanks for catching that!